btrack/napari/widgets/_general.py

```diff
@@ -79,32 +79,33 @@ def create_update_method_widgets() -> dict[str, tuple[str, QtWidgets.QWidget]]:
 
     max_search_radius = QtWidgets.QDoubleSpinBox()
     max_search_radius.setRange(0, 1000)
+    max_search_radius.setStepType(
+        QtWidgets.QAbstractSpinBox.AdaptiveDecimalStepType
+    )
     max_search_radius.setToolTip(
         "The local spatial search radius (isotropic, pixels) used when the update "
         "method is 'APPROXIMATE'"
     )
     max_search_radius.setWrapping(True)  # noqa: FBT003
-    max_search_radius.setStepType(
-        QtWidgets.QAbstractSpinBox.AdaptiveDecimalStepType
-    )
     widgets["max_search_radius"] = ("search radius", max_search_radius)
 
     max_lost_frames = QtWidgets.QSpinBox()
+    max_lost_frames.setRange(0, 10)
+    max_lost_frames.setStepType(
+        QtWidgets.QAbstractSpinBox.AdaptiveDecimalStepType
+    )
     max_lost_frames.setToolTip(
         "Number of frames without observation before marking as lost"
     )
     max_lost_frames.setValue(5)
-    max_lost_frames.setStepType(
-        QtWidgets.QAbstractSpinBox.AdaptiveDecimalStepType
-    )
     widgets["max_lost"] = ("max lost", max_lost_frames)
 
     not_assign = QtWidgets.QDoubleSpinBox()
-    not_assign.setToolTip("Default probability to not assign a track")
     not_assign.setDecimals(3)
-    not_assign.setValue(0.001)
     not_assign.setRange(0, 1)
     not_assign.setStepType(QtWidgets.QAbstractSpinBox.AdaptiveDecimalStepType)
+    not_assign.setToolTip("Default probability to not assign a track")
+    not_assign.setValue(0.001)
```

Comment:
Did we want to set this default to 0.1?

Reply:
I figured that would be done in quantumjot/btrack#374. If we merge this first then won't be an issue.